srcs/juloo.keyboard2/KeyModifier.java

```diff
@@ -297,6 +300,32 @@ private static KeyValue handleFn(KeyValue k, int flags)
       case "c": name = "¢"; break;
       case "p": name = "₱"; break;
       case "€": case "£": return removed_key; // Avoid showing these twice
+      // alternating greek letters
+      case "π": name = "ϖ"; break;
+      case "θ": name = "ϑ"; break;
+      case "Θ": name = "ϴ"; break;
```

Comment:
This is `Shift+Fn+θ`, quite complicated, these symbols are used as often as the greek letters, I think.
What about adding `ϑ` to the keyboard ? `ϴ` would be `Shift+ϑ`.

Same for the others `ϐ`, `ϱ`, etc...

Reply:
Like in my comment in the above conversation, ϴ is a variation of Θ. 
LaTeX and the desktop Bone-Layout don't have this variation, but I thought it would make sense to include it anyway.